composeApp/src/commonMain/kotlin/repository/TableBetsRepositoryImpl.kt

```diff
@@ -0,0 +1,24 @@
+package repository
+
+import koinModules.ApiClient
+import koinModules.`interface`.TableBetsRepository
+import kotlin.jvm.Synchronized
+
+class TableBetsRepositoryImpl(val apiClient: ApiClient): TableBetsRepository {
+    val tableBetsMap: HashMap<Int, Set<Int>> = hashMapOf()
+
+    @Synchronized
+    override fun getPlayedNumbersForTableId(tableId: Int): Set<Int> {
+        //TODO consider throwing here instead of returning an empty set
+        return tableBetsMap.get(tableId) ?: setOf()
```

Comment:
once we start throwing here, clicking teh action button will make the app crash every time since it doesn't have a valid tableId.

also, clicking on any game (boardId) will also probably throw and crash, so might be better to just get rid of this TODO instead eh?

Reply:
got rid of TODO